float64 m_timestamp

bool m_trackingSuccessful

pau2motors/fsQuaternionf m_headRotation
pau2motors/fsVector3f    m_headTranslation

float32 m_eyeGazeLeftPitch
float32 m_eyeGazeLeftYaw
float32 m_eyeGazeRightPitch
float32 m_eyeGazeRightYaw

float32[] m_coeffs

pau2motors/fsVector3f[] m_markers
